.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
         - pip install 'travis-cargo<0.2' --user && export PATH=$HOME/.local/bin:$PATH
       script:
         - cargo doc --no-deps --all-features
-        - cargo bench
+        - cargo test --benches
```

Comment:
I'm not familiar with `test --benches` -- does this automatically build in release?

Reply:
or does it just "run the benches once", i.e. treat them as tests.

Reply:
Yes, that's it. I don't see any value in running the actual benchmarks on CI hardware.

Reply:
Agreed. I didn't know about that flag, it's pretty cool! :smile: